### ISSUE TITLE
Voice avatars can't answer long answers with more than 1000 characters

### ISSUE THREAD

**Issue by applelogintest** [HeyGen-Official/StreamingAvatar#18]
Voice avatars can't answer long answers with more than 1000 characters

**Comment by MohitPanpaliya** [comment]
> Voice avatars can't answer long answers with more than 1000 characters

have you tried using different avatars for streaming

**Comment by vcpandya** [comment]
Yes that's the limit. But you should anyways send a sentence at a time, reducing the time lag and no risk of 1000 char limit
